doc/pingdom.md

```diff
@@ -0,0 +1,76 @@
+# Pingdom checks
+
+## Setting up the checks
+
+The checks are defined in terraform configuration file `terraform/pingdom/pingdom.tf`.
+
+### Requirements
+
+* Download or build the terraform-provider-pingdom binary. See below for instructions.
+* Make sure you have access to the PaaS credential store, this is required for Pingdom credentials.
+* Load the AWS credentials for the environment you are setting up checks for. These are required as the Terraform state file for the Pingdom checks is stored in an S3 bucket.
+
+### Usage
+
+#### Configure
+Configure the IDs of the contacts to be notified as a comma-delimited string in variable PINGDOM_CONTACT_IDS in the Makefile.
+
+#### Run
+Run `make <ENV> pingdom` to set up the Pingdom checks.
+
+## Use the binary
+Binaries for Linux and MacOS are available in our [paas-terraform-provider-pingdom repository](https://github.com/alphagov/paas-terraform-provider-pingdom/releases).
+
+Download and copy to a directory in your `$PATH`.
+
+## Build from source
```

Comment:
Could you make it a bit clearer that you would only want to do the following steps if you were developing changes to the provider or library? We want to guide people towards the pre-built binaries in normal circumstances, otherwise there's a chance that they'll waste time or use an incompatible version.